Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:fermium-bullseye AS BUILD_IMAGE
+FROM node:20.13.1-bullseye-slim AS BUILD_IMAGE
```

Comment:
Isn't this going to be a problem for (or conflict with) https://github.com/uber/cadence-web/blob/080605a8e64e31c917558ec4bdc4cd354d97839d/package.json#L33?

Reply:
Probably yes but We could update both with the same version of node instead 14.19.3 we upgrade to 20.13.1.

Reply:
We have some requests to upgrade node versions to node 18+, and this needs some testing efforts.

Reply:
@drjoeycadieux did you test the app using the newer version?

Reply:
I did not do it

Reply:
@Assem-Uber i would like to know if do package.json and packge-lock.json

Reply:
@drjoeycadieux 
Let's prepare the PR for merging by unifying the version across different files `Dockerfile`, `docker-compose.yml` & `package.json`. Also, the [engines.npm](https://github.com/uber/cadence-web/blob/080605a8e64e31c917558ec4bdc4cd354d97839d/package.json#L34) needs to be the version supported in`node:20.13.1-bullseye`

Reply:
i added a `help wanted` tag for testing the changes

Reply:
I did some changes in the files you mention.

Reply:
Do you have time to help out here @mantas-sidlauskas ?